Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonAttributeValue.java

```diff
@@ -74,9 +74,8 @@ void addInstance(AbstractCommonAttributeInstance metadata) {
         this.fileInstances.add(metadata);
     }
 
-    void addFileInstanceMetadata(AbstractCommonAttributeInstance metadata, String caseName) {
+    void addFileInstanceMetadata(AbstractCommonAttributeInstance metadata) {
```

Comment:
Do we need this method? Seems to do the same as addInstance().

Reply:
Looks like an auto-merge error or an oversight on our part.